adage/analyze/tests.py

```diff
@@ -789,30 +789,67 @@ def test_heavy_genes(self):
 
         self.call_non_get_API(uri)  # Test non-get methods too.
 
+    def create_extra_experiments(self):
+        """
+        Generate a few more experiements, one of which is returned
+        (and will be used by test cases later).
```

Comment:
Specifically, this function will create 3 random experiments in the Database using fixtureless and one experiment that is manually created, and will be returned, right? Maybe we should put this in the docstring?